cartographer_grpc/framework/rpc_handler_interface.h

```diff
@@ -28,6 +28,9 @@ class Rpc;
 class RpcHandlerInterface {
  public:
   virtual ~RpcHandlerInterface() = default;
+  // Returns the fully qualified name of the gRPC method this handler is
+  // implementing.
+  virtual std::string method_name() const = 0;
```

Comment:
Perhaps add a comment where one can look this up when implementing a new handler?

Reply:
Done.